mobile/src/card-view.js

```diff
@@ -560,7 +558,7 @@ export class CardListItem extends Component {
               {this.props.firstName} {this.props.lastName}
             </Text>
             <Text style={{ flexWrap: 'wrap', fontSize: 14, color: '#A8A8A8' }}>
-              {this.props.title}, {this.props.company}
+            { [this.props.title, this.props.company].filter(x => x).join(', ') }
```

Comment:
repeated 3 times... looks like a good candidate for refactoring into a function like:
```js
const titleAndCompany = (title, company) => [this.props.title, this.props.company].filter(x => x).join(', ')
```